### PR TITLE
strategy post for dripline for may

### PR DESCRIPTION
**Pull request by writerly03**
Please check that the post looks OK, including the image

### CODE REVIEW

File: _posts/2023-05-31-strategy-is.md
Comment (RangerMauve) [inline _posts/2023-05-31-strategy-is.md:73]:
```suggestion

---
```

Would a line break that separates the summary from the previous section be useful?

**Review by RangerMauve — APPROVED** [review]
Image is good, wording seems okay, added a small suggestion but I think this post is great without it too.

**Comment by tripledoublev** [comment]
This is an incredible post! I'm now merging into staging for ease of sharing and then creating the PR to bring it to production once we have everyone's approval

**Pull request merged**
